tasks/configure_nginx.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: copy ssl cert
+  copy:
+    content: "{{ DALITE_SSL_CERT }}"
+    dest: "/etc/ssl/private/ssl-cert.pem"
+    group: www-data
+    mode: "740"
+
+- name: copy ssl key
+  copy:
+    content: "{{ DALITE_SSL_KEY }}"
+    dest: "/etc/ssl/private/ssl-cert.key"
+    group: www-data
+    mode: "740"
```

Comment:
As of today nginx terminates SSL, and nginx is ran under `www-data:www-data` credentials. So this must stay. Possibly this should be changed to `440`.